### PR TITLE
Introducing a star (direct source) for clear sky simulations 

### PR DESCRIPTION
**Pull request by m-brath**

### CODE REVIEW

**Comment by olemke** [comment]
Anything left to do before merging this PR?

**Review by riclarsson — CHANGES_REQUESTED** [review]
Some small comments

File: src/m_gas_scattering.cc [hunk 1]
```diff
@@ -0,0 +1,239 @@
...
+  ===========================================================================*/
+
+/* Workspace method: Doxygen documentation will be auto-generated */
+void gas_scatteringOff(Index& gas_scattering_do,
+                       Agenda& gas_scattering_agenda,
+                       const Verbosity&) {
+  // set flag to False (default)
+  gas_scattering_do = 0;
+
+  gas_scattering_agenda = Agenda();
```
[the remark this answers is not in the document]
Comment (olemke) [reply src/m_gas_scattering.cc]: Fixed

File: src/m_refraction.cc [hunk 2]
```diff
@@ -300,6 +300,48 @@ void refr_index_airMicrowavesGeneral(
   refr_index_air_group += n;
 }
 
+void refr_index_waterVisibleNIR(GriddedField3& complex_refr_index,
```
Comment (riclarsson) [inline src/m_refraction.cc:303]: Should probably be named complex_refr_indexWaterVisibleNIR

Add real value limit to documentation in methods.cc

File: src/m_refraction.cc [hunk 3]
```diff
@@ -390,6 +432,29 @@ void complex_refr_indexIceMatzler06(GriddedField3& complex_refr_index,
   }
 }
 
+/* Workspace method: Doxygen documentation will be auto-generated */
+void complex_refr_indexTConstant(GriddedField3& complex_refr_index,
```
Comment (riclarsson) [inline src/m_refraction.cc:436]: The temperature seems to not be used locally by this function so it should probably be dropped from the name

File: src/workspace.cc [hunk 4]
```diff
@@ -1775,6 +1815,23 @@ void define_wsv_data() {
           "Agenda recalculating spectra and Jacobian for iterative inversion methods.\n"),
       GROUP("Agenda")));
 
+  wsv_data.push_back(WsvRecord(
+      NAME("in_los"),
```
Comment (riclarsson) [inline src/workspace.cc:1819]: Perhaps name these two as gas_scattering_los_in and gas_scattering_los_out

File: src/workspace.cc [hunk 5]
```diff
@@ -2746,6 +2823,23 @@ void define_wsv_data() {
           "*output_file_formatSetBinary*\n"),
       GROUP("String"), String{"ascii"}));
 
+  wsv_data.push_back(WsvRecord(
+      NAME("out_los"),
+      DESCRIPTION(
+          "Incoming line-of-sight.\n"
```
Comment (riclarsson) [inline src/workspace.cc:2829]: Copy pasta?

The physical use should probably be in the variable description

**Review by riclarsson — APPROVED** [review]
I think this can be merged now.  Great work :)

**Pull request merged**